pkgs/applications/audio/mpg123/default.nix

```diff
@@ -48,16 +51,16 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  postInstall = lib.optionalString withConplay ''
+  postInstall = lib.optionalString (!libOnly && withConplay) ''
     mkdir -p $conplay/bin
     mv scripts/conplay $conplay/bin/
   '';
 
-  preFixup = lib.optionalString withConplay ''
+  preFixup = lib.optionalString (!libOnly && withConplay) ''
     patchShebangs $conplay/bin/conplay
   '';
 
-  postFixup = lib.optionalString withConplay ''
+  postFixup = lib.optionalString (!libOnly && withConplay) ''
```

Comment:
How about adding an assert that con play can and should only be build when not only the library is build? This should avoid some of the changes.